docs/tool.md

```diff
@@ -0,0 +1,111 @@
+# The `homebase-lite` utility tool
+
+The `homebase-lite` utility tool, can be used to deploy, compile and document
+the smart contract easily.
+
+<!-- TOC START min:2 max:3 link:true asterisk:false update:true -->
+- [Prerequisites](#prerequisites)
+  - [Build tools](#build-tools)
+  - [Operating system](#operating-system)
+  - [Executables](#executables)
+- [How to get](#how-to-get)
+  - [Build instructions](#build-instructions)
+- [Usage](#usage)
+  - [Getting the Michelson code of the compiled contract](#getting-the-michelson-code-of-the-compiled-contract)
+  - [Getting the initial storage for the contract](#getting-the-initial-storage-for-the-contract)
+  - [Getting the contract documentation](#getting-the-contract-documentation)
+  - [Deploying the contract](#deploying-the-contract)
+<!-- TOC END -->
+
+## Prerequisites
+
+### Build tools
+
+If you want to build the tool from source you will need the
+[Haskell Tool Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
```

Comment:
I would say that this section actually belongs under the `How to get`->`Build instructions` one, given that it's ins't a prerequisite to run the tool if one downloads the static binary.

However I see that in that section `stack` is introduced already right away, so I would just remove this one.